src/pages/Architecture/index.js

```diff
@@ -121,50 +144,69 @@ function ArchitecturesSection() {
 
     const deleteArchitecture = data => {
         if (initialized) {
-            fetch(process.env.REACT_APP_API_BASE_URL + '/api/blueprints/' + data.id, {
-                method: 'delete',
-                headers: {
-                    'Content-Type': 'application/json',
-                    Authorization: initialized ? `Bearer ${keycloak?.token}` : undefined,
-                },
-            })
-                .then(response => response.json())
-                .then(data => {
-                    // if (data.success) {
-                    const updatedArchitectures = architectures.filter(card => card.id !== data.id);
-                    setArchitectures(updatedArchitectures);
-                    // } else {
-                    //     console.error('Error deleting card:', data.error);
-                    // }
+            if ((parentId = 'Admin' && location.pathname === '/architectures')) {
+                fetch(process.env.REACT_APP_API_BASE_URL + '/api/refArchs/' + data.id, {
+                    method: 'delete',
+                    headers: {
+                        'Content-Type': 'application/json',
+                        Authorization: initialized ? `Bearer ${keycloak?.token}` : undefined,
+                    },
+                })
+                    .then(response => response.json())
+                    .then(res => {
+                        console.log(architectures);
+                        const updatedArchitectures = architectures.filter(card => card.architecture_id !== data.id);
+                        setArchitectures(updatedArchitectures);
+                        setTotalArchitectures(updatedArchitectures.length);
+                    })
+                    .catch(error => console.error('Error deleting card:', error));
```

Comment:
deleting  card is not the correct sentence,
let's call it as "deleting ref. arch"